server/controllers/shows.js

```diff
@@ -0,0 +1,58 @@
+const { Show } = require('../models');
+
+const logger = require('../../config/winston');
+
+exports.getShows = (req, res, next) => Show.findAll({
+  attributes: { exclude: ['BandId', 'createdAt', 'updatedAt'] },
+})
+  .then((result) => {
+    res.send(result);
```

Comment:
Can you add add a conditional to handle empty query results to tell the client they are a 404 - Not Found error? Sung found this bug yesterday, if a query can't find a result but the query is formed correctly it just returns null and it won't throw an error.